tests/libwnbd_tests/scsi_helper.h

```diff
@@ -0,0 +1,17 @@
+#include "pch.h"
+#include <wnbd.h>
+
+class ScsiPassthroughHelper
+{
+private:
+	PWNBD_DISK WnbdDisk;
```

Comment:
WnbdDisk seems to be unused. That being considered, not sure if we really need a class.